### ISSUE TITLE
Mark active menu item with dynamic added submenu

### ISSUE THREAD

**Issue by Serge3d** [jeroennoten/Laravel-AdminLTE#509]
In config\adminlte.php:
```php
[
    'text' => 'companies',
    'key' => 'companies',
    'icon' => 'fas fa-fw fa-building',
],
```
In AppServiceProvider:
```php
$events->listen(BuildingMenu::class, function (BuildingMenu $event) {
    $items = Company::all()->map(function (Company $company) {
        return [
            'text' => $company['short_name'],
            'url' => route('companies.show', $company),
            'icon' => 'fas fa-fw fa-landmark',
        ];
    });
    $event->menu->addIn('companies', ...$items);
});
```
The 'companies' menu item has a submenu containing the active menu item (added in the AppServiceProvider), but is not marked as active.

![2](https://user-images.githubusercontent.com/37361800/77397618-02b3c500-6db7-11ea-972a-3b324ed9f844.png)

**Comment by Koozza** [comment]
I'm running into the same problem. Did you find a solution already?

**Comment by lacodimizer** [comment]
This fix was released with version 3.1.1 now :)
